Add X-Forwarded-Prefix to the migration guide (#11267)
traefik/traefik: docs/content/migration/v2.md

and a new option has been introduced to specify which ones could go through the middleware chain before being removed: `<entrypoint>.forwardedHeaders.connection`.

Please check out the [entrypoint forwarded headers connection option configuration](../routing/entrypoints.md#forwarded-headers) documentation.

## v2.11.14

### X-Forwarded-Prefix

> In `v2.11.14`, the `X-Forwarded-Prefix` header is now treated like other `X-Forwarded-*` headers, and is removed if it sent from an untrusted source.

Reviewer: ```suggestion
In `v2.11.14`, the `X-Forwarded-Prefix` header is now handled like the other `X-Forwarded-*` headers: Traefik removes it when it's sent from an untrusted source.
```
